pilot/cmd/pilot-agent/main.go

```diff
@@ -380,8 +383,9 @@ func init() {
 		"Proxy IP address. If not provided uses ${INSTANCE_IP} environment variable.")
 	proxyCmd.PersistentFlags().StringVar(&role.ID, "id", "",
 		"Proxy unique ID. If not provided uses ${POD_NAME}.${POD_NAMESPACE} from environment variables")
-	proxyCmd.PersistentFlags().StringVar(&role.DNSDomain, "domain", "",
+	proxyCmd.PersistentFlags().StringVar(&DNSDomain, "domain", "",
 		"DNS domain suffix. If not provided uses ${POD_NAMESPACE}.svc.cluster.local")
+	role.DNSDomains = append(role.DNSDomains, DNSDomain)
```

Comment:
can I just do an assign here rather than an append?

Reply:
Yes, `role.DNSDomains = []string{DNSDomain}` but this is not common.

Reply:
addressed.  Not sure why I asked that question; knew the answer - must have been tired when self reviewing :)